sources/packages/backend/apps/api/src/services/disbursement-schedule/disbursement-schedule-service.ts

```diff
@@ -138,10 +138,12 @@ export class DisbursementScheduleService extends RecordDataModelService<Disburse
         "application.applicationStatus",
         "application.id",
         "application.pirStatus",
+        "application.data",
```

Comment:
Do we need to return the entire data?

Reply:
`application.data` is a jsonb. I did some research about it and I don't think it's possible to retrieve only that property of the jsonb through TypeORM as far as I could learn about it.

Reply:
@andrepestana-aot there is a way to get a particular property of jsonb using query builder in type orm. 

![image](https://github.com/bcgov/SIMS/assets/54600590/55f023bf-fb2c-4c74-9a21-d19fc6f806aa)

But if we pick a property, then we should use `getRawOne` instead of getOne. 

Also this option cannot be used in typeorm find* query.

Reply:
Yes, @dheepak-aot I know that is possible with the `getRawOne`. I just didn't mention that because it doesn't apply here.